<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexnosisWindow</class>
 <widget class="QMainWindow" name="HexnosisWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hexnosis - Hex Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hexnosis.qrc">
    <normaloff>:/icons/hexacross.ico</normaloff>:/icons/hexacross.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRow_Shading"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionClose"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionRow_Shading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Row Shading</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="document-close">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="hexnosis.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>HexnosisWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
